tests/test_basic.py

```diff
@@ -1,6 +1,7 @@
 import os
 
-def test_basic ():
+
+def test_basic():
     cmd = "/usr/local/gcn2hop/gcn2hop --help"
```

Comment:
After the install target in the Makefile is changed appropriately, this `cmd` should work with `gcn2hop --help`.